setup.cfg

```diff
@@ -5,6 +5,9 @@ version = attr: things.__version__
 max_line_length = 88
 ignore = E501,E203,W503
 
+[pydocstyle]
+ignore = D102,D212,D213,D413,D203,D409,D405,D406,D407
```

Comment:
@mikez note that I disabled a large number of warnings - you can decide if there are issues that you feel are relevant:
```
./things/api.py:3 at module level:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:26 in public function `tasks` (skipping C901):
        D212: Multi-line docstring summary should start at the first line
./things/api.py:229 in public function `areas`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:229 in public function `areas`:
        D413: Missing blank line after last section ('Examples')
./things/api.py:299 in public function `tags`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:299 in public function `tags`:
        D413: Missing blank line after last section ('Examples')
./things/api.py:366 in public function `checklist_items`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:366 in public function `checklist_items`:
        D413: Missing blank line after last section ('Returns')
./things/api.py:392 in public function `search`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:392 in public function `search`:
        D413: Missing blank line after last section ('Examples')
./things/api.py:422 in public function `get`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:448 in public function `todos`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:457 in public function `projects`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:469 in public function `inbox`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:478 in public function `today`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:507 in public function `upcoming`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:519 in public function `anytime`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:528 in public function `someday`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:537 in public function `logbook`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:548 in public function `trash`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:565 in public function `canceled`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:574 in public function `completed`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:574 in public function `completed`:
        D413: Missing blank line after last section ('Examples')
./things/api.py:592 in public function `deadlines`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:603 in public function `last`:
        D212: Multi-line docstring summary should start at the first line
./things/api.py:643 in public function `token`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:131 in public class `Database`:
        D203: 1 blank line required before class docstring (found 0)
./things/database.py:131 in public class `Database`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:131 in public class `Database`:
        D413: Missing blank line after last section ('Parameters')
./things/database.py:553 in public function `dict_factory`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:553 in public function `dict_factory`:
        D405: Section name should be properly capitalized ('See Also', not 'See also')
./things/database.py:553 in public function `dict_factory`:
        D413: Missing blank line after last section ('See Also')
./things/database.py:553 in public function `dict_factory`:
        D407: Missing dashed underline after section ('See Also')
./things/database.py:553 in public function `dict_factory`:
        D406: Section name should end with a newline ('See Also', not 'See also:')
./things/database.py:571 in public function `escape_string`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:600 in public function `make_filter`:
        D213: Multi-line docstring summary should start at the second line
./things/database.py:600 in public function `make_filter`:
        D413: Missing blank line after last section ('Examples')
./things/database.py:628 in public function `make_date_filter`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:678 in public function `make_truthy_filter`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:678 in public function `make_truthy_filter`:
        D413: Missing blank line after last section ('Examples')
./things/database.py:707 in public function `make_search_filter`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:707 in public function `make_search_filter`:
        D413: Missing blank line after last section ('Example')
./things/database.py:707 in public function `make_search_filter`:
        D409: Section underline should match the length of its name (Expected 7 dashes in section 'Example', got 8)
./things/database.py:741 in public function `validate`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:741 in public function `validate`:
        D413: Missing blank line after last section ('Examples')
./things/database.py:770 in public function `validate_offset`:
        D212: Multi-line docstring summary should start at the first line
./things/database.py:770 in public function `validate_offset`:
        D413: Missing blank line after last section ('Examples')
tests/test_things.py:21 in public class `ThingsCase` (skipping V103 pylint: disable=R0904):
        D203: 1 blank line required before class docstring (found 0)
```